bridges/evm/contracts/example/DEX.sol

```diff
@@ -2,128 +2,113 @@ pragma solidity 0.5.14;
 pragma experimental ABIEncoderV2;
 
 import {SafeMath} from "openzeppelin-solidity/contracts/math/SafeMath.sol";
-
-interface Bridge {
-    struct VerifyOracleDataResult {
-        bytes data;
-        bytes32 codeHash;
-        bytes params;
-    }
-    function relayAndVerify(bytes calldata data)
-        external
-        returns (VerifyOracleDataResult memory result);
-}
+import { IBridge } from "../IBridge.sol";
 
 contract DEX {
-    using SafeMath for uint256;
-
-    bytes32 public codeHash;
-
-    mapping(address => mapping(bytes => uint256)) _balances;
-
-    Bridge bridge = Bridge(0x3e1F8745E4088443350121075828F119075ef641);
-
-    constructor(bytes32 _codeHash) public {
-        codeHash = _codeHash;
+  using SafeMath for uint256;
```

Comment:
Why is the change in this PR? I thought this is about updating LuckyNumber contract?